src/DataflowPass.cpp

```diff
@@ -51,10 +55,13 @@ void DataflowPass::transfer(Instruction *instruction,
             ProgramVariable assignedVar = ProgramVariable(store->getOperand(1));
             std::string arg = assignedVar.getCleanedName();
             PVAliasSet *pvas = inputProgramPoint.getPVASRef(assignedVar, false);
+            // this->onFunctionCall(pvas, fnName);
+            // TODO: Ask, wouldn't we want to keep track of functions like this
+            // if we want to make something that infers annotations?
```

Comment:
> wouldn't we want to keep track of functions like this if we want to make something that infers annotations?

Functions like what? At this point, we could be analyzing any function that is called on the RHS of an assignment.

In general, inferring annotations will definitely require us to reason about transfers like this one, but not just here - there will be many places where we'll need to make changes. I would prefer that we focus on getting a version of this system working where we assume that someone provides all of the annotations that we need first, though, before we worry about inference.